src/Context/AccesibilityContext.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Behatch\Context;
+
+use Behat\Gherkin\Node\TableNode;
+use Behat\Mink\Exception\ExpectationException;
+use Behat\Mink\Exception\ResponseTextException;
+use Behat\Mink\Exception\ElementNotFoundException;
+use WebDriver\Exception\StaleElementReference;
+use Behat\Behat\Tester\Exception\PendingException;
+
+class AccesibilityContext extends BaseContext
+{
+    /**
+     * @Then all images should have an alt attribute
+     */
+    public function allImagesShouldHaveAnAltAttribute()
+    {
+        $images = $this->getSession()->getPage()->findAll('xpath', '//img[not(@alt)]');
```

Comment:
Hi, could you tell me this would refer to test image without alt?
I pull your code and test my website. There are `alt` attribute, but still said `There are images without an alt attribute`